stdlib/public/core/UnsafeRawBufferPointer.swift.gyb

```diff
@@ -316,6 +316,41 @@ extension Unsafe${Mutable}RawBufferPointer: ${Mutable}Collection {
     }
     return 0
   }
+
+  public func _customIndexOfEquatableElement(_ element: UInt8) -> Index?? {
+    guard !isEmpty else { return Optional(nil) }
+
+    let ptr = baseAddress._unsafelyUnwrappedUnchecked
+    var i = 0
```

Comment:
For clarity, you should use `startIndex` here, and `endIndex` instead of `count` when comparing to `i`.

Reply:
fixed